CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Removed
+* `ui-message` component: attribute `title`.
+### Added
+* `ui-message` component: observer `_didVisibilityChange`.
```

Comment:
Observer _didVisibilityChange is private & information about component's inner structure is unnecessary in public changelog.
Observer was added to react on changes in `visible` property & send public actions `onShow` & `onHide`, so you can write about it in "Changed" paragraph instead of "Added": "Changed ui-message component: onShow & onHide actions now sent on every change in `visible` property".